gin-sample/adapter.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"github.com/gin-contrib/sessions"
+)
+
+type Storage struct {
+	session sessions.Session
```

Comment:
Wondering if `Session` is a built-in type

Reply:
`Session` is not provided by golang.
This `Session` comes from `github.com/gin-contrib/sessions`.